mod_reforged/hooks/entity/tactical/enemies/orc_berserker.nut

```diff
@@ -79,19 +79,19 @@
 	q.assignRandomEquipment = @(__original) function()
 	{
 	    __original();
+	    ::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 4);
+	}
 
-	    local weapon = this.getMainhandItem();
-	    if (weapon == null) return;
-
-	    ::Reforged.Skills.addPerkGroupOfEquippedWeapon(this, 6);
+	q.onSkillsUpdated = @(__original) function()
+	{
+		__original();
+		local weapon = this.getMainhandItem();
+		if (weapon == null) return;
```

Comment:
I'd personally do this the inverse way i.e. wrap the following in a `if (weapon != null)` block as it is more concise.